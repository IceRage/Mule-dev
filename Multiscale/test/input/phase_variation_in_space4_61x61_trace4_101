Area,Distance from origin,Angle
103,13.6015,18.0653
112.5,19.4165,38.8448
485.5,8.48528,66.6148
