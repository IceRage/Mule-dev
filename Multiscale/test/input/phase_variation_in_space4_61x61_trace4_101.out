Area,Distance from origin,Angle(degrees)
109,13.6015,30.564
109,18.6011,45
492.5,7.81025,67.3801
